confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -308,10 +308,9 @@ def get_compatibility(self, subject=None):
             url += '/' + subject
 
         result, code = self._send_request(url)
-        if code >= 200 and code <= 299:
+        if 200 <= code <= 299:
             compatibility = result.get('compatibility', None)
-
-        if not compatibility:
+        else:
```

Comment:
Don't think this is correct: result.get() may return None, in which case we should fall back to compatibilityLevel here.

Reply:
Yep, absolutely. Great catch, thanks